src/main/java/Parser.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Optional;
+
+public class Parser {
+    public static Optional<Command> parse(String input) {
+        String[] arr = input.split("\\s");
+        int index = 0;
+        switch (arr[0].toLowerCase()) {
+        case "bye":
+            if (arr.length > 1) {
+                Ui.printError(new Exception("A word of bye is enough"));
+            }
+            return Optional.of(new ExitCommand());
+
+        case "list":
```

Comment:
I personally would sort the cases in my switch case alphabetically (as it'll get easier to read, as we have to implement more and more features), but that's just a preference!

Reply:
I have sorted the switch cases as suggested.